site/docs/tutorial/android-app.md

```diff
@@ -396,6 +396,7 @@ installing it.
 
 For more details, see these pages:
 
+* View open feature requests and issues on [Bazel's GitHub](https://github.com/bazelbuild/bazel/issues)
```

Comment:
"Bazel's Github" sounds strange
How about a more common usage:
- Open issues and feature requests on [github](..url).